tests/0102-static_group_rebalance.c

```diff
@@ -800,15 +1103,12 @@ static void do_test_static_membership_mock(
 }
 
 int main_0102_static_group_rebalance(int argc, char **argv) {
-        /* TODO: check again when regexes
-         * will be supported by KIP-848 */
-        if (test_consumer_group_protocol_classic()) {
-                do_test_static_group_rebalance();
-        }
 
         if (test_consumer_group_protocol_classic()) {
+                do_test_static_group_rebalance_classic();
                 do_test_fenced_member_classic();
         } else {
+                do_test_static_group_rebalance_consumer();
```

Comment:
Let's add one more test for `UNRELEASED_INSTANCE_ID` error in the new protocol. This is a contract change.